.changeset/quiet-stars-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/ssg-plugins-essential': major
```

Comment:
I think `major` is okay, but in some modules, we specify it as `minor` for the first release. If you set this `major`, it will be released `1.0.0`. I think we don't need to release as `v1` first.